tests/unit/azure/test_credentials.py

```diff
@@ -542,3 +544,68 @@ def test_create_access_connectors_for_storage_accounts(sp_migration):
     assert len(validation_results) == 1
     assert validation_results[0].name.startswith("ac")
     assert len(validation_results[0].failures) == 0
+
+
+@pytest.fixture
+def sp_migration_1(installation, credential_manager):
```

Comment:
Don't create a fixture for something that is used only once, as it makes it harder to debug. Rewrite